packages/@aws-cdk/aws-ecr/lib/auth-token.ts

```diff
@@ -18,3 +20,27 @@ export class AuthorizationToken {
   private constructor() {
   }
 }
+
+/**
+ * Authorization token to access the global public ECR Gallery via Docker CLI.
+ *
+ * @see https://docs.aws.amazon.com/AmazonECR/latest/public/public-registries.html#public-registry-auth
+ */
+export class PublicGalleryAuthorizationToken {
```

Comment:
I considered exposing this in the existing `AuthorizationToken` using an optional `public: true`, but decided against it because the context of an authorization token for a private registry is region specific, while this refers to the global public gallery. 

I also think this makes for a clearer API, albeit a little less discoverable.